src/syseng/dump.448

```diff
@@ -8017,7 +8066,8 @@ READ9:	POPAE P,[TT,T,E,D,C,B]
 ;B - source index in BUFFER
 ;C - number of 4-word blocks
 ;Arguments get advanced, T,TT are clobbered
-RDCNV:	DMOVE T,BUFFER(B)	;First two 32-bit words
+RDCNV:	MOVE T,BUFFER(B)	;First 32-bit word
```

Comment:
I think this is brittle. If someone defines TT to be more than one register away from T it will break. Better to use T+1 in second MOVE.

Reply:
I understand your point, but it's not that simple.  The code below relies on it being TT.  But it's also doing LSHC and ROTC, so really TT must be T+1.

Reply:
Resolved by asserting TT=T+1 at assembly time.